ADOPTERS.md

```diff
@@ -1,18 +1,13 @@
----
-title: Adopters
-sidebar_position: 100
```

Comment:
do we really want to delete all those `sidebar_positions` - this defines the ordering int he sidebar, and ensures a somewhat good structure within the docusaurus page. else those will be ordered alphabetically.

Also to note, that docusaurus will use the highest one, as the default page for the section.
So if we click community in the header "Adopters" will no be displayed instead of the "Readme.md"

Reply:
i see what you did in the other repository :) the question is, if we might want to add this sidebar.js within here - as we might also include this community content into other documentation sources, i think it should be part of this repository rather than the other one.

Reply:
Yeah, that's a good idea. I'll try that out now. It's a bit more manual to maintain a sidebar but it allows us to avoid setting frontmatter on non-doc pages (e.g. the root README.md).